src/6_branch.js

```diff
@@ -1419,6 +1419,30 @@ Branch.prototype['sendSMS'] = wrap(
 	}
 );
 
+Branch.prototype['qrCode'] = wrap(
+	callback_params.CALLBACK_ERR_DATA,
+	function(done, linkData, qrCodeSettings, options) {
+		var data = utils.cleanLinkData(linkData);
+		data['qr_code_settings'] = safejson.stringify(utils.convertObjectValuesToString(qrCodeSettings || {}));
+		this._api(
+			resources.qrCode,
+			utils.cleanLinkData(linkData),
+			function(error, rawBuffer) {
+				var qrCodeResult = {};
+				if (!error && rawBuffer) {
+					qrCodeResult.rawBuffer = rawBuffer;
+					try {
+						qrCodeResult.base64 = btoa(String.fromCharCode.apply(null, new Uint8Array(rawBuffer)));
```

Comment:
These should be fine with IE 10+ even `Unit8Array` constructor, but I'll double-check

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCharCode
https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/btoa

Reply:
This is nice. The only thing you might consider before releasing this new API is making these into methods like `getBase64()` or just `base64()`. That could allow you to defer the base64-encoding until it's requested and provide more opportunity to extend in the future, like adding optional params. Just a suggestion/thought.

Reply:
that makes sense! I agree with return methods that way we do defer the base64 encoding

Reply:
added as a function. LMK if you have a better suggestion.